build-image/first-boot.bash

```diff
@@ -156,12 +152,12 @@ fi
 echo -n "$(timestamp) [openHABian] Ensuring network connectivity... "
 if ! running_in_docker && tryUntil "ping -c1 8.8.8.8 &> /dev/null || curl --silent --head http://www.openhab.org/docs |& grep -qs 'HTTP/1.1 200 OK'" 5 1; then
   echo "FAILED"
-  if grep -qs "openHABian" /etc/wpa_supplicant/wpa_supplicant.conf && iwconfig |& grep -qs "ESSID:off"; then
-    echo "$(timestamp) [openHABian] I was not able to connect to the configured Wi-Fi. Please check your signal quality. Reachable Wi-Fi networks are:"
-    iwlist wlan0 scanning | grep "ESSID" | sed 's/^\s*ESSID:/\t- /g'
-    echo "$(timestamp) [openHABian] Please try again with your correct SSID and password. The following Wi-Fi configuration was used:"
-    cat /etc/wpa_supplicant/wpa_supplicant.conf
-    rm -f /etc/wpa_supplicant/wpa_supplicant.conf
+
+  if [[ "$hotSpot" == "enable" ]] && ! [[ -x $(command -v comitup) ]]; then
+    #echo -n "$(timestamp) [openHABian] Installing comitup hotspot (will reboot after)... "
+    if setup_hotspot "install"; then echo "OK"; else echo "FAILED"; fi
+    #echo "$(timestamp) [openHABian] Hotspot software installed. Rebooting your system to make it take effect!"
+    #reboot
```

Comment:
Are these commented lines left here intentionally?

Reply:
sort of, yes
unsure if it works without rebooting but seems now it does